src/features/basicConvo.ts

```diff
@@ -1,12 +1,17 @@
 import { App } from "@slack/bolt";
+import PrettyError from "pretty-error";
+
+const pe = new PrettyError();
```

Comment:
Ao invés de adicionar em todos os arquivos, tu pode fazer ele prettyficar todos os erros de uma vez. o:
Isso evita ficar adicionando o mesmo trecho de código em vários arquivos e lugares diferentes.

Pra isso, teria só que adicionar no `src/index.ts` o trecho abaixo depois da parte que inicia o app. c:

```javascript
const pe = new PrettyError();
pe.start();
```

Reply:
Eu também tinha pensado nisso, até ler o troubleshooting da documentação.

https://github.com/AriaMinaei/pretty-error#troubleshooting

> `PrettyError.start()` modifies the stack traces of all errors thrown anywhere in your code, so it could potentially break packages that rely on node's original stack traces. I've only encountered this problem once, and it was with `BlueBird` when `Promise.longStackTraces()` was on.

> In order to avoid this problem, it's better to not use `PrettyError.start() `and instead, manually catch errors and render them with `PrettyError`